doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -266,3 +266,50 @@
         // perform error handling
     }
     ```
+
+14. Remove `forEach` from helplets
+    ```cpp
+    // before
+    iox::cxx::forEach(container, [&] (element) { /* do stuff with element */ });
+
+    // after
+    for (const auto& element: container) { /* do stuff with element */ }
+    ```
+
+15. Remove `enumTypeAsUnderlyingType`
+    ```cpp
+    constexpr const char* SOME_ENUM_STRINGS[] = {"FOO", "BAR"};
+
+    // before
+    std::cout << SOME_ENUM_STRINGS[iox::cxx::enumTypeAsUnderlyingType(someEnum)] << std::endl;
+
+    // after
+    std::cout << SOME_ENUM_STRINGS[static_cast<uint64_t>(someEnum)] << std::endl;
+    ```
+
+16. Remove `convertEnumToString`
+    ```cpp
+    constexpr const char* SOME_ENUM_STRINGS[] = {"FOO", "BAR"};
+
+    // before
+    std::cout << iox::cxx::convertEnumToString(SOME_ENUM_STRINGS, someEnum] << std::endl;
+
+    // after
+    std::cout << SOME_ENUM_STRINGS[static_cast<uint64_t>(someEnum)] << std::endl;
+    ```
+
+17. Replace `strlen2` with more generic `arrayCapacity`
+    ```cpp
+    constexpr const char LITERAL1[] {"FOO"};
+    constexpr const char LITERAL2[20] {"BAR"};
+    constexpr const uint32_t ARRAY[42] {};
+
+    // before
+    std::cout << iox::cxx::strlen2(LITERAL1) << std::endl; // prints 3
+    std::cout << iox::cxx::strlen2(LITERAL2) << std::endl; // prints 19
+
+    // after
+    std::cout << arrayCapacity(LITERAL1) << std::endl; // prints 4
```

Comment:
Yes, it is better to include the null terminator for string literals (`const char []`) like this.